ui/components/app/custom-spending-cap/custom-spending-cap-tooltip.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Box from '../../ui/box';
-import Typography from '../../ui/typography';
 import Tooltip from '../../ui/tooltip';
 import {
   TextColor,
   DISPLAY,
-  TypographyVariant,
+  TextVariant,
 } from '../../../helpers/constants/design-system';
-import { Icon, IconName, IconSize } from '../../component-library';
+import {
+  Icon,
+  ICON_NAMES,
+  ICON_SIZES,
+} from '../../component-library/icon/deprecated';
+import { Text } from '../../component-library';
```

Comment:
We are currently in the process of migrating to TypeScript we want to use the enums over the constants

```suggestion
import {
  Icon,
  IconName,
  IconSize,
  Text
} from '../../component-library';
```

Reply:
This may have some formatting issues that need linting